OOPBASICSPART1TASK6/src/com/gyk/java/oop/basics/TravelDestination.java

```diff
@@ -0,0 +1,53 @@
+package com.gyk.java.oop.basics;
+
+public class TravelDestination {
+    private String name;
+    private String city;
+    private int distance;
+
+    @Override

```

Comment:
equals and hasCode should be at the end of the class and not at the top

Reply:
ok